data/src/main/java/entities/JPAPOIEntity.java

```diff
@@ -0,0 +1,52 @@
+package entities;
+
+import enums.POITypes;
+
+import javax.persistence.*;
+
+
+@Entity
+@Table(name = "Object")
+public class JPAPOIEntity {
+
+    @Id
+    @Column(name = "id", length = 15, nullable = false)
+    private long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "X-crd")
+    private double X;
+
+    @Column(name = "Y-crd")
+    private double Y;
+
+    @Column(name = "Type")
+    @Enumerated(EnumType.STRING)
+    private POITypes type;
+
+    @Column(name = "Discription")
```

Comment:
Columns in database should start from lower case also - please fix in all places